account_statement_import_sheet_file/readme/CONTEXT.md

```diff
@@ -0,0 +1,15 @@
+BUSINESS NEED:
```

Comment:
I would suggest
```
Business need
~~~~~~~~~~~~~
```

The same for all titles

Reply:
Thanks for the idea! I'll take the suggestion to the functional group since the titles are not right now in the template, and we should change that first

Reply:
Well, you should compare how it is seen in both cases:

Old:

![Captura desde 2024-10-17 07-35-14](https://github.com/user-attachments/assets/3c5e70d0-6e19-4f88-9ef9-5e8bf8188174)

New:

![image](https://github.com/user-attachments/assets/1f754a41-13bf-436c-98dc-687d051b51fd)